boxsdk/client/client.py

```diff
@@ -1666,3 +1666,68 @@ def create_metadata_template(self, display_name, fields, template_key=None, hidd
             session=self._session,
             response_object=response,
         )
+
+    @api_call
+    def create_zip(self, name, items):
+        """
+        Creates a zip of multiple files and folders.
+
+        :param name:
+            The name of the zip file to be created.
+        :type name:
+            `unicode`
+        :param items:
+            List of files or folders to be part of the created zip.
+        :type items:
+            `Iterable`
+        :returns:
+            A dictionary representing a created zip
+        :rtype:
+            :class:`dict`
+        """
+        # pylint: disable=protected-access
+        url = self._session.get_url('zip_downloads')
+        data_items = []
+        for item in items:
+            data_items.append({'type': item._item_type, 'id': item.object_id})
+        data = {
+            'download_file_name': name,
+            'items': data_items
+        }
+        response = self._session.post(url, data=json.dumps(data)).json()
+        return self.translator.translate(
+            session=self._session,
+            response_object=response,
+        )
+
+    @api_call
+    def download_zip(self, name, items, writeable_stream):
+        """
+        Creates a zip of multiple files and folders and downloads it.
```

Comment:
```suggestion
        Downloads a zip file containing multiple files and/or folders.
```

Reply:
I think "create a zip" could be misleading because it might not be clear that the zip file is created "on Box."  Also, does the client really care that there is a creation step happening in the method?  I think they just care about the downloading of the zip file, so I removed the "creation part."